exercises/concept/little-sisters-vocab/.docs/introduction.md

```diff
@@ -0,0 +1,203 @@
+# Introduction
+
+A `str` in Python is an [immutable sequence][text sequence] of [Unicode code points][unicode code points].
+These could include letters, diacritical marks, positioning characters, numbers, currecy symbols, emoji, punctuation, space and line break characters, and more.
+Being immutable, a `str` object's value in memory doesn't change; methods that appear to modify a string return a new copy or instance of `str`.
+
+A `str` literal can be declared via single `'` or double `"` quotes. The escape `\` character is available as needed.
+
+```python
+
+>>> single_quoted = 'These allow "double quoting" without "escape" characters.'
+
+>>> double_quoted = "These allow embedded 'single quoting', so you don't have to use an 'escape' character".
+```
+
+Muliti-line strings are declared with `'''` or `"""`.
+
+```python
+>>> triple_quoted =  '''Three single quotes or "double quotes" in a row allow for multi-line string literals.
+  Line break characters, tabs and other whitespace is fully supported.
+
+  You\'ll most often encounter these as "doc strings" or "doc tests" written just below the first line of a function or class definition.
+    They\'re often used with auto documentation ✍ tools.
+    '''
+```
+
+Strings can be concatenated using the `+` operator.
+This method should be used sparingly, as it is not very performant or easily maintained.
+
+```python
+language = "Ukrainian"
+number = "nine"
+word = "девять"
+
+sentence = word + " " + "means" + " " + number + " in " + language + "."
+
+>>> print(sentence)
+...
+"девять means nine in Ukrainian."
+```
+
+If a `list`, `tuple`, `set` or other collection of individual strings needs to be combined into a single `str`, [`<str>.join(<iterable>)`][str-join], is a better option:
+
+```python
+# str.join() makes a new string from the iterables elements.
+>>> chickens = ["hen", "egg", "rooster"]
+>>> ' '.join(chickens)
+'hen egg rooster'
+
+# Any string can be used as the joining element.
+>>> ' :: '.join(chickens)
+'hen :: egg :: rooster'
+
+>>> ' 🌿 '.join(chickens)
+'hen 🌿 egg 🌿 rooster'
+```
+
+Code points within a `str` can be referenced by 0-based index number from the left:
+
+```python
+creative = '창의적인'
+
+>>> creative[0]
+'창'
+
+>>> creative[2]
+'적'
+
+>>> creative[3]
+'인'
+```
+
+Indexing also works from the right, starting with `-1`:
+
+```python
+creative = '창의적인'
+
+>>> creative[-4]
+'창'
+
+>>> creative[-2]
+'적'
+
+>>> creative[-1]
+'인'
+
+```
+
+There is no separate “character” or "rune" type in Python, so indexing a string produces a new `str` of length 1:
+
+```python
+
+>>> website = "exercism"
+>>> type(website[0])
+<class 'str'>
+
+>>> len(website[0])
+1
+
+>>> website[0] == website[0:1] == 'e'
+True
+```
+
+Substrings can be selected via _slice notation_, using [`<str>[<start>:stop:<step>]`][common sequence operations] to produce a new string.
```

Comment:
Maybe one example with a step, since you are mentioning it here

Reply:
Good point. 😄   I was worried it would overlap too much with  the sequence exercise -- but most of this is going to overlap anyways.  Part of the point of doing sequences as a separate exercise to begin with  is to show that str, list, and tuple have quite a lot in common.